ProductionSystem/Service/ProductionManagerHandler.py

```diff
@@ -17,11 +17,15 @@
 transTypes = [six.string_types, int, list]
 
 
-class ProductionManagerHandlerBase(RequestHandler):
```

Comment:
Are you sure about this change ? I guess the idea was that specific ProductionManagers in extensions can start from this base class.

Reply:
Well, TransformationManager is an example of an extended (by LHCb) database without the need for this trick.

Reply:
This is fine with me. I just always try to understand the developer reasoning behind this or that solution.

Reply:
Honestly I don't remember. If you think that there is no need for this it's also fine with me.